react-gbajs/package.json

```diff
@@ -13,7 +13,7 @@
     "@babel/polyfill": "7.7.0",
     "@babel/runtime": "7.7.2",
     "@babel/plugin-transform-runtime": "7.6.2",
-    "js-sha1": "0.6.0",
+    "lodash.clonedeep": "4.5.0",
```

Comment:
super minor/nitpicky, the lodash team says it's better to avoid these individual packages and use main lodash instead. If you do `import cloneDeep from 'lodash/cloneDeep'` it should only pull that into the bundle

https://lodash.com/per-method-packages